README.md

```diff
@@ -1,30 +1,52 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/zeit/next.js/tree/canary/packages/create-next-app).
-
-## Getting Started
-
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
+# Understanding vulnerability
+Allows staff to capture vulnerability and assets and have them surfaced in Single View.
+
+## Getting started
+This project uses **yarn** for dependency management and is built with Next.js.
+
+1. Install the project dependencies
+   ```
+   yarn
+   ```
+2. Create a `.env` file based off of the `.env.sample` that exists.
+3. [Set up DynamoDB Local](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html)
+4. Create local DynamoDB plans table
+   ```bash
+   aws dynamodb create-table --cli-input-json file://./config/tables/vulnerabilities.json --endpoint-url http://localhost:8000
+   ```
+5. Start running your local copy of understanding vulnerability.
+   ```
+   yarn dev
+   ```
+
+## Working with DynamoDB locally
+If you need to view, edit or delete data from your local copy of DynamoDB when working on a feature
+there is a useful admin tool you can run.
+
+```(bash)
+yarn dynamo-admin
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+## Testing
 
-## Learn More
+### Unit tests
+Unit tests are written with Jest, we are using `@testing-library/react` for React component testing.
 
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+```(bash)
+yarn unit-test
+```
 
-You can check out [the Next.js GitHub repository](https://github.com/zeit/next.js/) - your feedback and contributions are welcome!
+### Integration tests
+Integration tests are written with Cypress.
 
```

Comment:
Is it worth adding pages/Api routes description?

Reply:
Updated, thanks!